tools/ingest-converter/src/main/resources/ingest-pipeline.js

```diff
@@ -3,13 +3,28 @@
  */
 function ingest_pipeline_to_logstash(json) {
 
+    function handle_on_failure_pipeline(on_failure_json, tag_name) {
+
+        return IngestConverter.create_tag_conditional(tag_name,
+            IngestConverter.join_hash_fields(on_failure_json.map(map_processor))
+        );
+    }
+
     function map_processor(processor) {
 
         var filter_blocks = [];
         if (IngestGrok.has_grok(processor)) {
             filter_blocks.push(
                 IngestConverter.create_hash("grok", IngestGrok.grok_hash(processor))
```

Comment:
@suyograo I think we should just use `IngestGrok.get_name()` here as well, otherwise the fact this is supposed to be a constant becomes kinda obscure, right?